readded everything (#104)
MrLongbottom/P10-Article: sections/appendix/applications.tex

\subsection{Applications of our models}\label{sec:appendix_applications}
There are a variety of different applications that topic modeling could be used for. 
\citet{Probabilistic_Topic_Models} describes many different purposes for topic modeling, like exploring the history of news articles over time.
The \gls{lda} model, created by \citet{blei2003latent}, has seen many extensions over the years to try and improve the generality of the model.
Normally \gls{lda} works by inferring hidden topic structure, which is based on two distributions, namely the document-topic and topic-word distributions.
An extension to the \gls{lda} is the Author-Topic model, created by \citet{author_topic_2012}, which creates a relationship between the author meta-information and the corpus.

> This idea of taking meta-information into account seems to be overlooked, even though a few papers have touched upon this area, the application possibilities for these kind of meta-information integrations are endless.

Reviewer: ```suggestion
This idea of taking meta-information into account seems to be overlooked, even though a few papers have touched upon this area, the application possibilities for these kinds of meta-information integrations are endless.
```

> News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling can help improve their processes such as searching, recommendation, the grouping of articles, information completion, etc.

Reviewer: ```suggestion
News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling could help improve their processes such as searching, recommendation, the grouping of articles, information completion, etc.
```

Author: ```suggestion
News media groups, like Nordjyske, are trying to find new ways of integrating smarter and more intelligent methods for keeping their customers, and using topic modeling could help improve their processes such as searching, recommendation, grouping of articles, and information completion.
```

We will briefly give an overview of each of these to explain how topic modeling might play a role in improving these processes.

The first example is to improve search functionality for their articles.
This problem could be alleviated by using the topics created by our topic models for performing query expansions finding similar words to the ones that appear in the query, providing more context to find results from.
Once initial search results have been found, topic models could also be used to find other results with similar topics.
These techniques can be particularly useful as they can help find articles that do not use any of the words in a given query but are still relevant to the query, which is a property that most basic search algorithms do not possess.

Another very important problem within news agencies is the recommendation of articles.

> Topic models can be used particularly for content-based filtering, finding other articles with similar topics either to users' preferences based on their history, or other similar articles.

Author: ```suggestion
Topic models can be used particularly for content-based filtering, finding other articles with similar topic distribution, either based on a users preferences represented by a interaction history or based on a specific article being read.
```

Reviewer 2: ```suggestion
Topic models can be used particularly for content-based filtering, finding other articles with similar topic distributions, either based on a user's preferences represented by an interaction history or based on a specific article being read.
```

> Grouping together items, or clustering, can serve many different purposes. 

Reviewer 2: ```suggestion
Grouping items together, or clustering, can serve many different purposes. 
```

Topic modeling provides a new way of grouping together articles based on topic similarity.

A topic model might also be used to fill in missing meta information in an article dataset.

> For example, with the taxonomy-topic model, we sample new taxonomies for the majority of the dataset, which does not already have a taxonomy entry.

Reviewer 2: Passer det? For dokumenter uden taxonomy får vi modellen til at estimere ud fra teksten at det f.eks. er "STEDER/Danmark/..."?
